src/shared-components/chip/style.ts

```diff
@@ -14,7 +14,7 @@ export const ChipText = styled.span`
 `;
 
 const defaultStyle = css`
-  background-color: ${COLORS.purple10};
+  background-color: ${COLORS.disabled};
```

Comment:
```suggestion
  background-color: ${COLORS.defaultBackground};
```

They're the same value, but semantically `chip` component can't be disabled, so I think we should prefer `defaultBackground`.